packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -101,7 +101,7 @@ const Item = ({
       </div>
       <div className="grid grid-cols-3 items-center gap-x-4">
         <div className="flex items-center justify-end gap-x-4">
-          <Text size="small">
+          <Text size="small" style={{ paddingTop: 3 }}>
```

Comment:
Q: Can't we use tailwind here (and below) for the padding?

Reply:
yep 👍